frontend/admin/src/js/components/pool/PoolTable.tsx

```diff
@@ -125,7 +126,8 @@ export const PoolTable: React.FC<GetPoolsQuery & { editUrlRoot: string }> = ({
           id: "ioqFVF",
           description: "Title displayed for the Pool table status column.",
         }),
-        accessor: "status",
+        accessor: ({ advertisementStatus }) =>
+          intl.formatMessage(getAdvertisementStatus(advertisementStatus || "")),
```

Comment:
I think this null coalesce isn't really useful.  If it's null and coalesces to an empty string the getAdvertisementStatus will throw an error anyway since that key doesn't exist.  If you do a test first you could "else" fallback to https://github.com/GCTC-NTGC/gc-digital-talent/blob/c36788f0f61bcd1da87aaa355c906773128c635d/frontend/common/src/messages/commonMessages.ts#L24 maybe?